### PR TITLE
Examples: Add `glsl` tag and format `fragmentShader` with `tab`

### PR DESCRIPTION
**Pull request by linbingquan**
Related issue: #XXXX

**Description**

Add `glsl` tag for `fragmentShader`.

Format `fragmentShader` with `tab`.

### CODE REVIEW

File: examples/jsm/shaders/FXAAShader.js [hunk 1]
```diff
@@ -32,255 +32,255 @@ const FXAAShader = {
...
+		// PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+		// PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+		// OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+		// (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+		// OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+		//
+		//----------------------------------------------------------------------------------
+
+		#ifndef FXAA_DISCARD
+				//
```
Comment (Mugen87) [inline examples/jsm/shaders/FXAAShader.js:81]: I think the indentation uses one tab too much.

Comment (linbingquan) [reply examples/jsm/shaders/FXAAShader.js]: > I think the indentation uses one tab too much.

Done. My mistake...  just simply using tab...

Comment (Mugen87) [reply examples/jsm/shaders/FXAAShader.js]: Unfortunately, the entire file is affected by this issue. I think you can easier see this if you enable configure your editor such that whitespaces are displayed.

File: examples/jsm/shaders/FXAAShader.js [hunk 2]
```diff
@@ -156,46 +156,46 @@ const FXAAShader = {
 
 			// 45 deg edge detection and corners of objects, aka V/H contrast is too similar
 			if( abs( relativeVContrast ) < .3 ) {
```
Comment (linbingquan) [inline examples/jsm/shaders/FXAAShader.js:158]: This comparison isn't very intuitive, I just changed tab...

**Comment by linbingquan** [comment]
@Mugen87 There is a piece of code comparison is not intuitive( I added comment ), other should be no problem, please review again. Thank you.

File: examples/jsm/shaders/FXAAShader.js (hunk 2 above)
Comment (Mugen87) [reply examples/jsm/shaders/FXAAShader.js:158]: I wouldn't change the code at all since it is from a reference implementation.

**Pull request merged**